os/arch/arm/src/amebasmart/hardware/amebasmart_memorymap.h

```diff
@@ -93,14 +95,14 @@
  * are not known apriori and must be specified with configuration settings.
  */
 
-#define AMEBASMART_ROMCP_SECSIZE          (1024*1024) /* 0x00000000 - 0x00100000  1MB Boot ROM */
-#define AMEBASMART_FLASH_SECSIZE          MKULONG(128*1024*1024) /* 0x08000000 - 0x10000000  128MB Flash */
-#define AMEBASMART_SRAM_SECSIZE           MKULONG(256*1024*1024) /* 0x20000000 - 0x40000000  256MB SRAM */
-#define AMEBASMART_MMAP_DEV0_SECSIZE      MKULONG(256*1024*1024) /* 0x40000000 - 0x60000000  256MB Memory mapped device */
-//#define AMEBASMART_NS_SECSIZE             (size_t)__text_end__ - (size_t)__text_start__ /* __text_start__ - __text_end__  MB NS code */
-//#define AMEBASMART_NON_CACHE_SECSIZE      (size_t)__ram_nocache_end__ - (size_t)__ram_nocache_start__ /* __ram_nocache_start__ - __ram_nocache_end__  MB Non-cache code */
-#define AMEBASMART_DRAM_SECSIZE           MKULONG(256*1024*1024)  /* 0x60000000 - 0x80000000  256MB Other DRAM */
-#define AMEBASMART_MMAP_DEV1_SECSIZE      MKULONG(2048*1024*1024) /* 0x80000000 - 0xFFFFFFFF  2048MB Memory mapped device */
+#define AMEBASMART_ROMCP_SECSIZE          (1024*1024)             /* 0x00000000 - 0x00100000  1MB Boot ROM */
```

Comment:
How about defining calculated value?
i.e. 1048576 instead of (1024*1024)

Because `1024*1024` causes multiply in code operation.

You can leave `1024*1024` at a code comment.

Reply:
done~